lib/model/Manufacturer.js

```diff
@@ -15,7 +16,20 @@ module.exports = class Manufacturer {
     return manufacturers[this.key].website || null;
   }
 
+  get hasComment() {
```

Comment:
Just a minor issue: We used to write the "has"-Getter after the data getter (e.g. `Fixture.hasComment` after `Fixture.comment` or `Channel.hasDefaultValue` after `Channel.defaultValue`) and IMO, we should stay consistent with that practice.